docs/index.md

```diff
@@ -6,27 +6,8 @@ ScalarDB is a universal transaction manager that achieves:
 - database/storage-agnostic ACID transactions in a scalable manner even if an underlying database or storage is not ACID-compliant.
 - multi-storage/database/service ACID transactions that can span multiple (possibly different) databases, storages, and services.
 
-## Install
```

Comment:
Maybe we don't need to update this `index.md` for now, except for the typo.

Reply:
Agreed. I think we can leave it as it is (except for fixing typos) since it will affect the current GitHub pages and `index.md` will be eventually deleted once we stop using GitHub pages for this repo.

Reply:
I'm OK with leaving `index.md` as is for the same reason @feeblefakie mentions.